.gitignore

```diff
@@ -13,8 +13,18 @@
 !/log/.keep
 !/tmp/.keep
 
+# Ignore uploaded files in development
+/storage/*
```

Comment:
can we not upload files to s3 in development? that's what we've done historically...which I like because they behave like remote files

Reply:
We can. But the default is `/storage` and I didn't change that for now. IMHO it's better to configure that on a per project basis, especially since the whole point of the `ActiveStorage` abstraction is that everything will behave the same way, no matter where it's stored.

Reply:
@hiattp Any thoughts?

Reply:
Ehhhhhhhh not really any useful thoughts on this...I like the fact that we're going with the default, but I'm skeptical of the "abstraction that works the same everywhere" theory (I like to guarantee that by actually using the same strategy everywhere). So I guess I'd prefer to stick with our current practice and not use local in dev but s3 elsewhere...but I'll defer to your judgment here.

Reply:
Point taken, but I think people can configure that on a per-project basis. At least with this default they can start working even before a client has provided S3 access etc and then just change the config later.